pkg/web_assets/static/styles.css

```diff
@@ -3,4 +3,13 @@
   src: url('/static/GeistMonoVariableVF.woff2') format('woff2');
   font-weight: 100 900;
   font-style: normal;
+}
+#version {
```

Comment:
we should use tailwind to style this, not custom css

Reply:
"mt-4 p-2 bg-slate-800 text-slate-300 text-center rounded-md text-sm"